grid_map_demos/launch/octomap_to_gridmap_demo_launch.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration
+from launch_ros.actions import Node
+
+
+def generate_launch_description():
+    grid_map_demos_dir = get_package_share_directory('grid_map_demos')
+
+    param_file = LaunchConfiguration('param_file')
+    rviz_config_file = LaunchConfiguration('rviz_config')
+
+    declare_param_file_cmd = DeclareLaunchArgument(
+        'param_file',
+        default_value=os.path.join(
+            grid_map_demos_dir, 'config', 'octomap_to_gridmap_demo.yaml'),
+        description='Full path to the config file to use')
+
+    declare_rviz_config_file_cmd = DeclareLaunchArgument(
+        'rviz_config',
+        default_value=os.path.join(
+            grid_map_demos_dir, 'rviz', 'octomap_to_gridmap_demo.rviz'),
+        description='Full path to the RVIZ config file to use')
+
+    # TODO(marwan99): re-enable once octomap_mapping is ported to ROS2
```

Comment:
So can this part of things not be used until octomap is ported? Any other things that we're still waiting for?

Reply:
We need to document that somewhere

Reply:
Yes, this launch file shouldn't be used now. The OctomapToGridmapDemo.cpp node could be used on its own though.

That's the only thing we are waiting for. Will it be okay to merge without waiting and sort this later once an octomap_mapping port is ready?

Shall I create an issue to document this? I was also thinking of commenting the whole file to avoid any potential confusion for anyone trying to use this launch file before its ready.

Reply:
Issue and/or readme